_resources/port1.0/group/app-1.0.tcl

```diff
@@ -126,6 +126,17 @@ proc app.get_default_identifier {} {
 options app.retina
 default app.retina no
 
+
+# app.dark_mode: whether the app supports dark mode
+#
+# The default is yes.
+#
+# Info.plist key NSRequiresAquaSystemAppearance.
+
+options app.dark_mode
+default app.dark_mode yes
```

Comment:
It would appear that to match the other similar retina option that @ryandesign added, the default for this should be "no" and it should be turned on if needed. Is there a reason to do it differently like this? Otherwise it will just get changed to "no" a few minutes after we commit it.

Reply:
I know and this is my thought:
all applications should support HighDPi (aka retina) and probably most dark-mode otherwise the text is quite unreadable and the 2D objects are blurry.
Therefore, from my point of view, those options should be always `yes` and changed to `no` case by case and only if it is really impossible to fix the problem.

Reply:
For this "app PortGroup" the usual role is to have a double-clickable front on a GTK or other unix application. Just for my education, do any of those kinds of X11/GTK3/etc applications have the capability to support these added features like "retina" or "dark mode"?

Reply:
wxWidgets might support some of them.

Reply:
I tested the following:
- fltk works with retina; dark-mode not tested
- wxwidgets: retina works; dark-mode present on 3.2 (aka 3.1)
- gtk2/gtk3+quartz: retina works; dark-mode seems to work but require more tests
- qt: retina works (in some situation macports version requires more power than upstream bundle probably little bug because upstream use MACOSX_DEPLOYMENT_TARGET 10.9 or 10.11); dark mode works